### DISCUSSION TITLE
[Unstable] submarine flooding for no reason, but only when mirrored for pvp

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#15419] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

https://github.com/user-attachments/assets/5947ddd6-e2ed-4d53-a04a-c4214ae1148f

[SvS beacon vs beacon.zip](https://github.com/user-attachments/files/18216066/SvS.beacon.vs.beacon.zip)

now that the bad mirroring position has been fixed, it works great, however now this is happening (only on the mirrored side)

### Reproduction steps

1. use my file
2. start sub v sub with it on both sides
3. why is the hissihiss sinking????????????????????????

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### -

new unstable with the alien ruin pvp map

### Version

Other

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next update. Closing discussion.
